pkg/kubelet/kubelet.go

```diff
@@ -297,6 +297,7 @@ func PreInitRuntimeService(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 
 	switch containerRuntime {
 	case kubetypes.DockerContainerRuntime:
+		klog.Warningf("Using Docker is deprecated, please consider using a full-fledged CRI implementation")
```

Comment:
thanks, should we allude to the fact that it's not being actively maintained anymore (re would help make issues like  https://github.com/kubernetes/kubernetes/pull/90839 go by faster :)

Reply:
indicating the built-in dockershim implementation is deprecated would be clearer... using docker via a CRI impl could still be reasonable, right?

Reply:
Ack @liggitt